tock/hours/models.py

```diff
@@ -222,6 +232,18 @@ class Timecard(models.Model):
                                             default=0.80, decimal_places=2, max_digits=3,
                                             verbose_name="Percentage of hours which are expected to be billable this week")
 
+    # Utilization reporting
```

Comment:
I love how well these are differentiated. Should they be non-editable though?

Reply:
Makes sense since the intention is to derive all of these attributes from other fields!